samples/variants/build.gradle

```diff
@@ -1,6 +1,6 @@
 apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
-//apply plugin: 'com.datadoghq.dd-sdk-android-gradle-plugin'
+apply plugin: 'com.datadoghq.dd-sdk-android-gradle-plugin'
```

Comment:
We need to comment it back, otherwise plugin won't be found (we have corresponding `classpath` entry commented in the root `build.gradle`).